app/views/katello/api/v2/activation_keys/base.json.rabl

```diff
@@ -3,16 +3,29 @@ extends 'katello/api/v2/common/timestamps'
 
 attributes :id, :name, :description, :unlimited_hosts, :auto_attach
 
-attributes :content_view_id
+node :content_view_id do |ak|
+  ak.single_content_view&.id
+end
+
+node :content_view do |ak|
+  ak.single_content_view&.slice(:id, :name)
+end
 
-child :content_view => :content_view do
+child :content_views => :content_views do
   attributes :id, :name
 end
 
-child :environment => :environment do
-  attributes :name, :id
+node :environment_id do |ak|
+  ak.single_lifecycle_environment&.id
+end
+
+node :environment do |ak|
+  ak.single_lifecycle_environment&.slice(:id, :name)
+end
+
+child :lifecycle_environments => :lifecycle_environments do
+  attributes :id, :name
```

Comment:
Lists of lifecycle environments without their corresponding CVs (and vice versa) are a leftover from when my multi-CV thinking was not quite as developed as it is now. I don't think they are needed. Instead we should just list single cv/lce for backward compatibility, and then also give the full list of `content_view_environments`.

Reply:
dont know if it will break anything. Nervous about changing an API response

Reply:
Isn't this code new?

Reply:
Oh right lol

Reply:
updated